packages/xrpl/src/Wallet/rfc1751.ts

```diff
@@ -176,15 +178,34 @@ function bufferToArray(buf: Buffer): number[] {
  * @param buf - A 128-bit (16 byte) buffer
  * @returns A buffer containing the same data with reversed endianness
  */
-function swap128(buf: Buffer): Buffer {
-  // Interprets buffer as an array of (two, in this case) 64-bit numbers and swaps byte order in-place.
-  const reversedBytes = buf.swap64()
-
-  // Swap the two 64-bit numbers since our buffer is 128 bits.
-  return Buffer.concat(
-    [reversedBytes.slice(8, 16), reversedBytes.slice(0, 8)],
-    16,
-  )
+function swap128(buf: Uint8Array): Uint8Array {
+  // // Interprets buffer as an array of (two, in this case) 64-bit numbers and swaps byte order in-place.
+  const reversedBytes = swap64(buf)
+  // // Swap the two 64-bit numbers since our buffer is 128 bits.
+  return concat([reversedBytes.slice(8, 16), reversedBytes.slice(0, 8)])
+}
+
+function swap(b: Uint8Array, n: number, m: number): void {
+  const i = b[n]
+  // eslint-disable-next-line no-param-reassign -- we have to swap
+  b[n] = b[m]
+  // eslint-disable-next-line no-param-reassign -- see above
+  b[m] = i
+}
+
+function swap64(arr: Uint8Array): Uint8Array {
+  const len = arr.length
+
+  if (len < 192) {
```

Comment:
I'm confused where this 192 number comes from, it doesn't seem to be from the 64, and doesn't match the name of the function.

Reply:
I derived the code from this.  https://github.com/nodejs/node/blob/95b198911382b0aa0a0448e2a86214558665e02b/lib/buffer.js#L1198-L1215. I am now seeing the issue though.  I just return it non swapped if it is greater than 192.

Reply:
Resolved by 2d66c38a06eb47f13102054958b599bace783b9d